migrations/20240510181122_store_nft_img_ipfs.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+-- +goose StatementEnd
+
+SET search_path = devices_api, public;
+ALTER TABLE user_devices ADD COLUMN ipfs_cid VARCHAR;
```

Comment:
Maybe name it `image_cid` or `ipfs_image_cid`? I could see it being a little ambiguous.